db/dcrsqlite/dcrsqlite_test.go

```diff
@@ -0,0 +1,83 @@
+package dcrsqlite
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+)
+
+// Clears target folder content
+func removeContents(dir string) error {
+	d, err := os.Open(dir)
+	if err != nil {
+		return err
+	}
+	defer d.Close()
+	names, err := d.Readdirnames(-1)
+	if err != nil {
+		return err
+	}
+	for _, name := range names {
+		err = os.RemoveAll(filepath.Join(dir, name))
+		if err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+const (
+	defaultDataDirname = "test.data"
+	defaultDBFileName  = "test.dcrdata.sqlt.db"
+)
+
+// Returns path of a temporary directory used by these tests to store some data
+func getTempFolderPath(t *testing.T) string {
+	testDir, err := filepath.Abs(defaultDataDirname)
+	if err != nil {
+		t.Fatalf("Failed to produce DB-test folder path: %v", err)
+	}
+	return testDir
+}
+
+// Ensures we run our test in a clean room. Removes all files created by any of these tests in the temp directory.
+func resetTempFolder(t *testing.T) {
+	testFolderPath := getTempFolderPath(t)
+
+	removeContents(testFolderPath)
+	err := os.RemoveAll(testFolderPath)
```

Comment:
Why remove contents manually with `removeContents` when `RemoveAll` does it for you?